frontend/src/actions/taskActions.js

```diff
@@ -272,6 +290,30 @@ const updateTask = task => {
   }
 }
 
+const deleteTask = (task, history) => {
+  validToken()
+  return dispatch => {
+    dispatch(deleteTaskRequested())
+    axios
```

Comment:
```suggestion
    return axios
```
If we return the axios call here we will have a promise that we will be able to process on the component and then call history, without the need to pass as parameter.